src/components/modebar/manage.js

```diff
@@ -88,52 +93,49 @@ function getButtonGroups(gd, buttonsToRemove, buttonsToAdd) {
         groups.push(out);
     }
 
+    function appendButtonsToAdd(groups) {
```

Comment:
This has a weird signature. Functions with `append____` usually take as argument *the thing to be appended*. 

Could we either rename this, or add the buttons array as an argument for readability?

`function appendButtonsToGroup(buttons, group)`